Web/Pages/CCGMember/AllPositions.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@page 
+@inject IViewLocalizer Localizer
+@model Web.Pages.CCGMember.AllPositionsModel
+@{
+    var isEnglish = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName == "en";
+    var jobGroups = Model.JobGroups;
+    var imagefolder = "~/img/CCGMember/";
+}
+<section role="contentinfo">
+    <div class="col-md-12">
+        <h2><b>@Localizer["AllPositions"]</b></h2>
+        <form>
+            <p>
+                <input type="text" asp-for="Filter" />
+                <input type="submit" text=@Localizer["Keywords"] value=@Localizer["Filter"] />
+            </p>
+        </form>
+        <form method="post">
```

Comment:
Shouldn't need the form for printing out the position details since there aren't form controls being used.